content/go/concepts/math-functions/terms/hypot/hypot.md

```diff
@@ -0,0 +1,82 @@
+---
+Title: 'Hypot()'
+Description: 'Returns the square root of the sum of two squares.'
+Subjects: 
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags: 
+  - 'Math'
+  - 'Arithmatic'
+  - 'Functions'
+CatalogContent: 
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Hypot()`** function returns the square root of the sum of two squares. In other words, the method returns the result of the following formula:
+
+hypotenuse = $`\sqrt{(x * x) + (y * y)}`$
+
+## Syntax
+
+```pseudo 
+  import math
+
+  result = math.Hypot(x, y)
+```
+
+`Hypot()` accepts two arguments (`x` , `y`) of type `float64` and returns a value of type `float64`.
+
+Note:
+- If either argument is negative, or both, the result is positive.
+- If either argument is positive infinity (`+Inf`), the result will be positive infinity.
+- If `0` is passed as both arguments, the result is `0`. If one of the arguments is `0` the result is equal to the other argument.
+- If either argument passed is `NaN`, the result will be `NaN`.
+
+
+## Example
+
+```go
```

Comment:
The entire code block should be shifted left and the indentations should use 2 spaces, tabs are not used in docs